### DISCUSSION TITLE
Moving channels into categories event

### DISCUSSION THREAD

**Discussion by mikemaster98** [discordjs/discord.js#5288] [Q&A] [answered]
I am coding a bot for a Discord channel, in which new channels are frequently added and moved. However, my problem lies in moving the channels between categories. Is there an event coupled with this? Something similar to `client.on('channelCreate')` or` client.on('channelDelete')`  but for the location of a channel. I cannot find anything on the discord.js info page.

**Comment by TwilightZebby** [comment, marked as answer]
[Discord's API Docs](https://discord.com/developers/docs/topics/gateway#commands-and-events-gateway-events) don't list an Event triggered for *specifically* when a Channel is moved - maybe its included in the `channelUpdate` Event? [D.JS Docs Link](https://discord.js.org/#/docs/main/12.5.1/class/Client?scrollTo=e-channelUpdate)

**Reply by mikemaster98** [reply]
That's what I thought. I will mess around for a bit, see if it does anything.

**Comment by mikemaster98** [comment]
It is included in the channelUpdate event. A simple 
`if (old_channel.parent.id != new_channel.parent.id)`

did wonders.
